src/routes/staff.js

```diff
@@ -0,0 +1,169 @@
+import express from 'express';
+import sql from 'mssql';
+import {createDecodedData} from '../utils/common.js';
+import {encode, decode} from '../utils/codification.js';
+import {dateStringParser} from "../utils/parsers.js";
+import {sqlResponseHandler} from "../utils/handlers.js";
+
+const ROUTER = express.Router();
+
+function generatePublicStaff(el){
+  return{
```

Comment:
Leave whitespace between the return and the curly 
```suggestion
  return {
```